CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+## Installation
+
+Install prerequisite tools:
+
+1. Node.js (install v18) https://nodejs.org/en
+   (Recommended to use a node manager like fnm or nvm)
+2. pnpm
+
+   ```sh
+   npm install --global pnpm
+   ```
+
+3. Cargo https://doc.rust-lang.org/cargo/getting-started/installation.html
+
+   ```sh
+   curl https://sh.rustup.rs -sSf | sh
+   ```
```

Comment:
nit: I think it would make sense to have this as step 4 possibly since the previous 2 & docker are more related to running the indexer

Reply:
We should revisit the file since it's very outdated